sdks/python/apache_beam/runners/portability/local_job_service_main.py

```diff
@@ -129,6 +131,9 @@ def run(argv):
     print('Server started at port', port)
     return
 
+  if options.default_pickler:
+    pickler.set_library(options.default_pickler)
```

Comment:
should we set the `--pickler_library` pipeline option in this case?

Also noting the naming difference for a similar concept - should this be renamed into `--pickler_library`?

Reply:
I wasn't aware that there was such a pipeline option (and it doesn't seem possible to set on a per-pipeline basis). 

If you feel it's worth changing the name, we should do so before the next release, though we'd have to make sure it gets set in both places (setting in stager seems too late).

Reply:
> I wasn't aware that there was such a pipeline option
It's used for example here: https://github.com/apache/beam/blob/f0cd27596e3a59d6ee013ee2db232d24f46ad70a/sdks/python/apache_beam/runners/worker/sdk_worker_main.py#L103

> it doesn't seem possible to set on a per-pipeline basis
I wonder if this is the same as: https://github.com/apache/beam/issues/21615

> If you feel it's worth changing the name, we should do so before the next release
I think it would be worthwhile. Would someone working on xlang have bandwidth look into that?

If we are making cloudpickle default, I would consider tightening the bounds on cloudpickle to 
cloudpickle>=2.2.0,<2.3.0 , and bumping that bound every beam release until we decide to vendor the library.

Reply:
I can take on this change. Should we just re-use `--pickle_library` like the pipeline option? (Actually fixing #21615 is out of scope for now.) It'd be another cherry-pick into the release branch.

Tightening the bounds feels like it should be a separate PR.

Reply:
https://github.com/apache/beam/pull/23111

Reply:
>  It'd be another cherry-pick into the release branch.

Cherrypick, assuming #23111 is merged as is: https://github.com/apache/beam/pull/23118

Reply:
> Tightening the bounds feels like it should be a separate PR.

Sending https://github.com/apache/beam/pull/23120